inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -526,6 +529,132 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"LSTMSequence"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                                 const std::map<std::string, std::string> params) -> CNNLayerPtr {
```

Comment:
May I ask you to pass this map by reference?

And fix CreatorFor functor.

Reply:
done in a separate PR https://github.com/openvinotoolkit/openvino/pull/1928